blockchain/validator/Node.go

```diff
@@ -59,7 +59,7 @@ func (app *Node) Initialize(configFile string, workingDir string, key ed25519.Pr
 
 	laddr := viper.GetString("rpc.laddr")
 
-	rpcClient, _ := rpchttp.New(laddr, "/websocket")
+	rpcClient, _ := rpchttp.New(laddr)
```

Comment:
`rpchttp.New` no longer accepts a second parameter. There is another function that accepts options, but the path option defaults to `/websocket` so we don't need to use it.